popsycle/run.py

```diff
@@ -95,7 +95,14 @@ def run():
     optional.add_argument('--overwrite',
                           help="Overwrite all output files.",
                           action='store_true')
-    args = parser.parse_args()
+
+
+    optional.add_argument('--pbh-config-filename', type=str,
+                      help='Name of configuration file containing '
+                           'pbh inputs. Default if needed is: '
+                           'pbh_config.yaml',
+                      default='pbh_config.yaml')
```

Comment:
Is this argument has a default value, then the condition on Line 186 (if args.pbh_config_filename is not None) will always be True.  Let's remove the default here.